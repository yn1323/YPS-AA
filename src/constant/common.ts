export const isCSR = typeof window !== 'undefined' || process.env.csr
export const csrLang = process.env.lang ?? 'ja'
export const FOOTER_LINKS = [
  { text: 'よくある質問', url: 'http://www.google.com', rel: 'noopener' },
  { text: 'お問い合わせ', url: 'http://www.google.com', rel: 'noopener' },
  {
    text: 'Illustrations by Storyset.',
    url: 'https://storyset.com/work',
    rel: 'noopener',
  },
]
export const TIME_UNIT = [
  {
    label: '5分毎',
    value: '5',
  },
  {
    label: '10分毎',
    value: '10',
  },
  {
    label: '15分毎',
    value: '15',
  },
  {
    label: '20分毎',
    value: '20',
  },
  {
    label: '30分毎',
    value: '30',
  },
]
export const SUBMIT_FREQUENCY = [
  {
    label: '1週間毎',
    value: '1w',
  },
  {
    label: '2週間毎',
    value: '2w',
  },
  {
    label: '1ヶ月ごと',
    value: '1m',
  },
]
