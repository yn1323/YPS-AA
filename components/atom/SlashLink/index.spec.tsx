import { shallow } from 'enzyme'
import React from 'react'
import Component from '.'
import { FOOTER_LINKS } from '@constant/common'
import { mountWithTheme, ThemeProvideWrapper } from '@test/TestWrapper'

test(`${Component.name}-snapshot`, () => {
  const wrapper = mountWithTheme(<Component links={FOOTER_LINKS} />)
  expect(wrapper).toMatchSnapshot()
})
