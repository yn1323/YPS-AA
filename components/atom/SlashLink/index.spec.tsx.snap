// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SlashLink-snapshot 1`] = `
<SlashLink
  links={
    Array [
      Object {
        "rel": "noopener",
        "text": "よくある質問",
        "url": "http://www.google.com",
      },
      Object {
        "rel": "noopener",
        "text": "お問い合わせ",
        "url": "http://www.google.com",
      },
      Object {
        "rel": "noopener",
        "text": "Illustrations by Storyset.",
        "url": "https://storyset.com/work",
      },
    ]
  }
>
  <SlashLink__Container>
    <div
      className="SlashLink__Container-sc-12n56qr-0 cgAmef"
    >
      <SlashLink__UrlLink
        href="http://www.google.com"
        key="0"
        rel="noopener"
      >
        <a
          className="SlashLink__UrlLink-sc-12n56qr-1 gBDHey"
          href="http://www.google.com"
          rel="noopener"
        >
          よくある質問
        </a>
      </SlashLink__UrlLink>
      <SlashLink__UrlLink
        href="http://www.google.com"
        key="1"
        rel="noopener"
      >
        <a
          className="SlashLink__UrlLink-sc-12n56qr-1 gBDHey"
          href="http://www.google.com"
          rel="noopener"
        >
          お問い合わせ
        </a>
      </SlashLink__UrlLink>
      <SlashLink__UrlLink
        href="https://storyset.com/work"
        key="2"
        rel="noopener"
      >
        <a
          className="SlashLink__UrlLink-sc-12n56qr-1 gBDHey"
          href="https://storyset.com/work"
          rel="noopener"
        >
          Illustrations by Storyset.
        </a>
      </SlashLink__UrlLink>
    </div>
  </SlashLink__Container>
</SlashLink>
`;
